### ISSUE TITLE
ESD mitigation for users, who confuse screen to being a "touch screen" 

### ISSUE THREAD

**Issue by khwas** [khwas/UA315#13]
- This is "LOW priority issue"
  ESD mitigation for users, who confuse screen to being a "touch screen".
- Study the existing solutions for proper ESD protection of TFT screen
- Consider adding controlled impedance path from TFT to metal panel and circuit ground (1KV 1000pF cap in parallel with 1MOhm resistor). To do so, choose most appropriate TFT pin or group of pins
- Or add high value resistors (and caps if possible) from all pins to ground
- Or add low value resistors in series to all pins (?) of TFT subassembly
